src/main/java/fi/hsl/transitdata/tripupdate/processing/TripUpdateProcessor.java

```diff
@@ -178,7 +178,14 @@ private TripUpdate updateTripUpdateCacheWithCancellation(final String cacheKey,
             List<StopTimeUpdate> stopTimeUpdates = getStopTimeUpdates(cacheKey);
             // We need to clean up the "raw data" StopTimeUpdates for any inconsistencies
             List<StopTimeUpdate> validated = GtfsRtValidator.cleanStopTimeUpdates(stopTimeUpdates, null);
-            builder.addAllStopTimeUpdate(validated);
+            if (validated.isEmpty()) {
```

Comment:
Maybe add a comment here to explain why this is done?

Reply:
Added comments